provisioning_templates/script/windows_default_script.erb

```diff
@@ -61,7 +61,7 @@ echo Writing install image to partition while downloading additional files
    start /min cmd /C "echo Downloading finsh script and activating SetupComplete.cmd
                 md c:\deploy
                 %WGET% --no-verbose <%= foreman_url("finish") -%> -O C:\deploy\foreman-finish.bat"
-) | pause
+) | pause >nul
```

Comment:
Suppress the 'Press any key to continue' message
![foreman_win_fix_script](https://user-images.githubusercontent.com/13570034/80579462-b3049100-8a27-11ea-84f7-95c1c151cda1.png)